src/components/Payroll/PayrollHistory/PayrollHistoryPresentation.tsx

```diff
@@ -0,0 +1,168 @@
+import { useTranslation } from 'react-i18next'
+import styles from './PayrollHistoryPresentation.module.scss'
+import { DataView, Flex } from '@/components/Common'
+import { useComponentContext } from '@/contexts/ComponentAdapter/useComponentContext'
+import { useI18n } from '@/i18n'
+import { HamburgerMenu } from '@/components/Common/HamburgerMenu'
+import { formatNumberAsCurrency } from '@/helpers/formattedStrings'
+import ListIcon from '@/assets/icons/list.svg?react'
+import TrashcanIcon from '@/assets/icons/trashcan.svg?react'
+
+export interface PayrollHistoryItem {
+  id: string
+  payPeriod: string
+  type: 'Regular' | 'Off-cycle' | 'Dismissal'
+  payDate: string
+  status: 'Unprocessed' | 'Submitted' | 'Pending' | 'Paid' | 'Complete' | 'In progress'
```

Comment:
Fine for now since this is presentation, but i'd recommend getting these types from the api once we properly configure this. also the type prop will always be regular for now until we start dealing with other payroll types